src/main/scala/com/datasonnet/Mapper.scala

```diff
@@ -90,32 +91,60 @@ object Mapper {
     (name -> jsonnetLibrary)
   }
 
+  def getParams(name: String, mimeType: String, header: Header): util.Map[String, Object] = {
+    if (header != null) {
+      val dfParams = header.getDataFormatParameters().get(mimeType);
+      if (dfParams != null) {
+        val mergedParams = dfParams.getOrDefault(Header.DATAFORMAT_DEFAULT, new util.HashMap[String, Object]())
+        mergedParams.putAll(dfParams.getOrDefault(name, new util.HashMap[String, Object]()))
```

Comment:
This will modify the defaults that were just retrieved

Reply:
Correct, that's what I wanted. First we load all default settings and then overrides. Imagine this - you have payload and 100 CSV variables. All but one have separator | and the only one has ,. So I should be able to do something like:

`dataformat.application/csv.*.Separator=|`
`dataformat.application/csv.weirdoVar.Separator=,`